### PR TITLE
added some descriptive information on results page

### PR DESCRIPTION
**Pull request by MAvecillas23**
explains what the air pollution and earthquake sections do and what they return.

### CODE REVIEW

File: static/styles.css [hunk 1]
```diff
@@ -69,3 +69,12 @@ body {
     background-color: var(--header-fg-color);
     border: 2px solid var(--btn-border-color);
 }
+
+#results-title {
+    color: var(--header-fg-color);
+    text-align: center;
```
Comment (littlestripes) [inline static/styles.css:75]: Setting this produced some weird output. Probably better just to leave it left-aligned. I can try to use some Flexbox to arrange it a little more deliberately if we have time.

Captura de imagem_20241104_110807

**Review by littlestripes — COMMENTED** [review]
See my comment on alignment, otherwise this will do fine.

File: static/styles.css (hunk 1 above)
Comment (MAvecillas23) [reply static/styles.css:75]: Agreed, if you can make it look a lot cleaner, please do. I removed the center align.

**Comment by MAvecillas23** [comment]
talked about in zoom meeting, all agreed good to merge

**Pull request merged**